.circleci/config.yml

```diff
@@ -392,6 +404,7 @@ workflows:
     jobs:
       - license-check
       - test-crds
+      - run-unit-tests
```

Comment:
This job has been added in the "operator" workflow although the unit tests could execute tests that are related to templates too. Might it be better to create another workflow called "common" where execute them?